Doc/library/stdtypes.rst

```diff
@@ -4822,35 +4822,66 @@ Generic Alias Type
    object: GenericAlias
    pair: Generic; Alias
 
-``GenericAlias`` objects are created by subscripting a class (usually a
-container), such as ``list[int]``.  They are intended primarily for
-:term:`type annotations <annotation>`.
-
-Usually, the :ref:`subscription <subscriptions>` of container objects calls the
-method :meth:`__getitem__` of the object.  However, the subscription of some
-containers' classes may call the classmethod :meth:`__class_getitem__` of the
-class instead. The classmethod :meth:`__class_getitem__` should return a
-``GenericAlias`` object.
+``GenericAlias`` objects are generally created by
+:ref:`subscripting<subscriptions>` a class. They are most often used with
+container classes (e.g. :class:`list` or :class:`dict`). They are intended
```

Comment:
I wanted to link to a definition of the term "container", but it doesn't seem to be in the glossary.

Reply:
Maybe a good one would be https://docs.python.org/3/reference/datamodel.html#emulating-container-types.

Reply:
Also I don't suggest writing ``They`` more than once to avoid ambiguity.

In ``They are intended``, a newbie might have no clue what's the ``They`` here refers to. ``GenericAlias`` or container classes? It doesn't hurt to repeat ``GenericAliases`` again.

Reply:
> Maybe a good one would be https://docs.python.org/3/reference/datamodel.html#emulating-container-types.

That'll do for now!